Fabric.Terminology.SqlServer/Persistence/SqlValueSetRepository.cs

```diff
@@ -129,15 +129,15 @@ public IValueSet GetValueSet(string valueSetId, IEnumerable<string> codeSystemCo
         }
 
         public Task<PagedCollection<IValueSet>> FindValueSetsAsync(
-            string nameFilterText,
+            string filterText,
             IPagerSettings pagerSettings,
             IEnumerable<string> codeSystemCodes,
             bool includeAllValueSetCodes = false)
         {
             var dtos = this.DbSet.Where(GetBaseExpression());
-            if (!nameFilterText.IsNullOrWhiteSpace())
+            if (!filterText.IsNullOrWhiteSpace())
             {
-                dtos = dtos.Where(dto => dto.ValueSetNM.Contains(nameFilterText));
+                dtos = dtos.Where(dto => dto.ValueSetNM.Contains(filterText) || dto.ValueSetOID.Contains(filterText));
```

Comment:
One thing that could boost performance without having to have the search term exactly equal the OID might be to use StartsWith and if no results are returned then search by name?